src/smart-components/group/groups.js

```diff
@@ -70,7 +70,7 @@ const Groups = () => {
           }}
         />
       </Route>
-      <Route exact path={paths['group-edit']}>
+      <Route exact path={ paths['group-edit'].path }>
```

Comment:
Remove the extra spaces so lint is fixed and we are good to go!
```suggestion
      <Route exact path={paths['group-edit'].path}>
```